### ISSUE TITLE
We talk about `ocfl_1.0.txt` but the spec is HTML with external dependencies, so what does that mean?

### ISSUE THREAD

**Issue by zimeon** [OCFL/spec#203]
In https://ocfl.io/draft/implementation-notes/#storage-root-hierarchy , for example, we talk about storing a copy of the spec as `ocfl_1.0.txt`. But the spec is HTML with external dependencies, so what does that mean?

**Comment by neilsjefferies** [comment]
Good point - it would be useful if the web-page build scripts could also produce a downloadable version (whatever that is) suitable for insertion into Storage Roots which could be linked from the main page.

**Comment by neilsjefferies** [comment]
This kind of means we should keep a versioned collection of these.

**Comment by neilsjefferies** [comment]
PDF is worse than the html since it's not readily command-line viewable.

**Comment by zimeon** [comment]
Agreement in [2018-11-21 editors' meeting](https://github.com/OCFL/spec/wiki/2018.11.21-Editors-Meeting) that we make a more general statement that the root MAY include a copy of the specification, use `ocfl_1.0.html` in the example
